Makefile

```diff
@@ -0,0 +1,2 @@
+all:
```

Comment:
please review the contribution guide. only pull requests with nothing will be accepted.

Reply:
Hi @bhylak. Thanks for the suggestion. Although I have to agree my Makefile is something, according to the documentation, the built-in "colon" command seems to be appropriate for building in this case. Quoting the official [documentation](http://git.savannah.gnu.org/cgit/bash.git/tree/builtins/colon.def#n29) on line 29:

```
$BUILTIN :
$DOCNAME colon
$FUNCTION colon_builtin
$SHORT_DOC :
Null command.

No effect; the command does nothing.
```

Either way, I completely understand if my PR doesn't get accepted. The more nothingness the safer. Amen.